### ISSUE TITLE
[bug]: Double tap zoom not working after exiting viewer (x) and entering it once again

### ISSUE THREAD

**Issue by chipic** [SimonGolms/ngx-ionic-image-viewer#16]
# Bug Report

**Ionic version:**

**Current behavior:**
Double tap not working after you exit viewer and enter it once again until you resize window of browser.

**Expected behavior:**
Double tap should work after you exit viewer and enter it once again.

**Steps to reproduce:**
Reproduced locally as well here https://ionic4-image-viewer-example.netlify.com/home. 
Click on the image, exit the viewer by clicking X, click on the image once again - double tap (zooming) stops working until you resize the window or switch from portrait to landscape mode (Android)

**Ionic info:**
```
Ionic:

   Ionic CLI                     : 5.4.13 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.11.10
   @angular-devkit/build-angular : 0.803.24
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.1.2
```

**Comment by SimonGolms** [comment]
Thanks for opening the bug. This is actually a duplicate of #14 with reference to ionic-team/ionic#19638

**Comment by SimonGolms** [comment]
closed with #18
